znai-cli/src/main/java/com/twosigma/znai/cli/ZnaiCliApp.java

```diff
@@ -153,13 +157,25 @@ private void generateDocs() {
                 withFooterPath(config.getSourceRoot().resolve("footer.md")).
                 withExtensionsDefPath(config.getSourceRoot().resolve("extensions.json")).
                 withWebResources(favIconResource).
+                withPageModifiedTimeStrategy(pageModifiedTimeStrategy()).
                 withEnabledPreview(config.isPreviewMode());
 
         this.webSite = config.isExportMode() ?
                 webSiteCfg.parseOnly():
                 webSiteCfg.deployTo(deployPath);
     }
 
+    private PageModifiedTimeStrategy pageModifiedTimeStrategy() {
+        switch (config.getModifiedTimeStrategy()) {
+            case FILE:
+                return new FileBasedPageModifiedTime();
+            case CONSTANT:
+                return new ConstantPageModifiedTime(Instant.now());
+            default:
+                return null;
```

Comment:
Why don't you return new FileBasedPageModifiedTime() as default?
If you do, you don't have to check null and create FileBasedPageModifiedTime() from 95@Website.java.

Reply:
Since there are other users of `WebSite` outside of `znai` I was thinking about making this parameter optional on that level.